commands/verify/verify.go

```diff
@@ -228,20 +274,33 @@ func DoVerifyAccordingConfig() error {
 
 			for current := 1; current <= retryCount; current++ {
 				if err := verifySingleCase(v.GetExpected(), v.GetActual(), v.Query); err == nil {
+					summary.successNum++
+					_ = spinnerLiveText.Stop()
 					break
 				} else if current != retryCount {
-					logger.Log.Warnf("verify case failure, will continue retry, %v", err)
+					if current == 1 {
+						logger.Log.Warnf("verify case[%d] failure, will continue retry", idx+1)
+					}
+					Msg := fmt.Sprintf("Retrying to verify case[%d]  [%d/%d]", idx+1, current, retryCount)
```

Comment:
And here, please also check other places

Reply:
sorry, I missed. QAQ